hbw/app/assets/stylesheets/hbw.sass

```diff
@@ -267,6 +267,32 @@
   table.disabled
     background-color: #eeeeee
 
+  .attacher
+    width: 150px
+    margin-top: 5px
+    padding: 8px
+    border-radius: 7px
+    border: 1px solid #ccc
+    color: #ccc
+    display: flex
```

Comment:
I recommend you to use some ordering rules for css properties.
Look at this https://9elements.com/css-rule-order/